### ISSUE TITLE
Generator: add an option to skip partitions

### ISSUE THREAD

**Issue by MassimilianoPoggiQonto** [go-jet/jet#437]
Currently if you have partitioned tables, with partitions residing on the same schema as your tables, `jet` will generate schemas for each partition individually.

It would be great if an `--ignore-partitions` flag could be added so that partitions are not included for generation.

**Comment by go-jet** [comment]
Different developers use various naming templates for partitions, so there's no single universal solution to exclude partition tables from generation.
You can [customize generator](https://github.com/go-jet/jet/wiki/Generator#generator-customization) to skip unwanted tables.

**Comment by MassimilianoPoggiQonto** [comment]
MSSql can rely on `sys.partitions` and Postgres can rely on `pg_class` and `pg_inherits` to identify partitions.

**Comment by go-jet** [comment]
Yeah, it seems you're right. We could add generator flag to handle this use case.

**Comment by lkzcover** [comment]
I’m trying to do it in the custom `postgres.GenerateDSN()` method. It works for the model package, but I haven’t understood how to skip partitions in the table package yet, because I can’t find a way to filter `schemaMetadata` in `template.ProcessSchema` yet.

Example:

```
func isPartitionTable(db *sql.DB, tableName string) bool {
	var count int
	err := db.QueryRow(`
		SELECT COUNT(*)
		FROM pg_inherits
		JOIN pg_class c ON pg_inherits.inhrelid = c.oid
		WHERE c.relname = $1`, tableName).Scan(&count)

	if err != nil {
		log.Fatal(err.Error())
	}
	return count > 0
}

err = postgres.GenerateDSN(
	dsn,
	schema,
	path,
	template.Default(postgres2.Dialect).
		UseSchema(func(schemaMetaData metadata.Schema) template.Schema {
			return template.DefaultSchema(schemaMetaData).
				UseModel(template.DefaultModel().
					UseTable(func(table metadata.Table) template.TableModel {
						if isPartitionTable(db, table.Name) {
							return template.TableModel{Skip: true}
						}
						...
					}),
				)
		}),
)
```

**Comment by sadensmol** [comment]
it would be great to add wildcard support for `ignore-tables` option, thus we could configure it for skippiing partitions when needed.
